src/utils/execute_async.php

```diff
@@ -6,7 +6,7 @@
 
 type SubprocessOptions = shape(
   ?'working_directory' => string,
-  ?'environment' => dict<string, string>,
+  ?'environment' => dict<string, arraykey>,
```

Comment:
What's the reason here? Environment variables are strings at the process level, which seems relevant here

Reply:
argh ,sorry ,was in your comment

Reply:
I discovered this by accident, but try var_dump()ing $_ENV. It will end with:
```
  ["GLIBCXX_FORCE_NEW"]=>
  string(1) "1"
  ["UNW_RBP_ALWAYS_VALID"]=>
  string(1) "1"
  ["HPHP"]=>
  int(1)
  ["HHVM"]=>
  int(1)
  ["HHVM_JIT"]=>
  int(1)
  ["HHVM_ARCH"]=>
  string(3) "x64"
}
```

`1` can be both a string and an int.
I am afraid that this is some PHPish behavior...